### PR TITLE
Adding CVE-2018-6487

### PR DESCRIPTION
**Pull request by cyberpsrt**

### CODE REVIEW

File: 2018/6xxx/CVE-2018-6487.json [hunk 1]
```diff
@@ -1,18 +1,87 @@
 {
```
Comment (cve-team) [inline 2018/6xxx/CVE-2018-6487.json]: JSON: u'version_value' is a required property

**Comment by cve-team** [comment]
Auto-validation for this pull request has detected issues:
JSON: Please see review comments for errors.

**Comment by cve-team** [comment]
Thank you for your submission. To clarify on the above auto-vaildation error, it appears you put, "version_name", instead of "version_value" as a field name. Making this change should allow you to pass the validator.

File: 2018/6xxx/CVE-2018-6487.json (hunk 1 above)
Comment (cyberpsrt) [reply 2018/6xxx/CVE-2018-6487.json]: The value was changed. Would I need to do a new pull request or can we continue on with this pull

**Comment by cve-team** [comment]
WARNING: Commit signed, but not with a known GPG key for this account. Please provide your GPG key id to the CVE team to be assoicated with this account, or check to ensure you have used the correct key.

**Comment by cyberpsrt** [comment]
Hello Team,



The value was changed within the branch. Would I need to do a new pull request or can we still do it with the same pull?



Kind Regards,



Micro Focus PSRT

**Comment by cyberpsrt** [comment]
The changes were made. Please let me know if there is anything else that should be done.

**Comment by cve-team** [comment]
Thank you for your submission, expect your entry to be published to cve.mitre.org in the next few hours.

Going forward, note that if there are any issues with the PR, you can choose how you wish to respond depending on what's easier: both closing out the PR, and updating the PR with new commits are acceptable.

**Pull request merged**